### ISSUE TITLE
A question about learning rate multipler for parameters.

### ISSUE THREAD

**Issue by wanderingpj** [dmlc/mxnet#4077]
In optimizer.py, there is a function named set_lr_mult( ) , it's for Seting individual learning rate multipler for parameters. And whether we should set this depends on the network(sym). Is that means different network gives parameters different learning rate?
https://github.com/dmlc/mxnet/blob/master/python/mxnet/optimizer.py#L130

**Comment by CNevd** [comment]
yes

**Comment by wanderingpj** [comment]
Why should we give parameters different learning rate? @CNevd

**Comment by CNevd** [comment]
sometimes we need different learning rate in different layers

**Comment by wanderingpj** [comment]
Got it , thank you! @CNevd

**Comment by kernel8liang** [comment]
learning rate is a hyper parameter, hyper parameters's construct a space each time you try to lean something you pick a point in that space, then train your model. So, different learning rate is very important.
